apps/aecore/src/aec_conductor.erl

```diff
@@ -1221,6 +1222,7 @@ handle_successfully_added_block(Block, Hash, true, PrevKeyHeader, Events, State,
     maybe_publish_tx_events(Events, Hash, Origin),
     maybe_publish_block(Origin, Block),
     State1 = maybe_consensus_change(State, Block),
+    io:format(user, "~nAdded Block: ~p~n",[Block]),
```

Comment:
I guess `lager:info/2` is being used in other parts of the code.

Reply:
🙈 Sorry (let's agree that you haven’t seen this sh...t )